rgi.xml

```diff
@@ -1,133 +1,158 @@
-<tool id="rgi" name="Resistance Gene Identifier (RGI)" version="3.2.0">
-    <description>This tool predict resistome(s) from protein or nucleotide data based on homology and SNP models.</description>
+<tool id="rgi" name="Resistance Gene Identifier (RGI)" version="4.2.2">
+    <description>This tool predicts resistome(s) from protein or nucleotide data based on homology and SNP models.</description>
     <requirements>
-        <requirement type="package" version="2.7">python</requirement>
-        <requirement type="package" version="3.2.0">rgi</requirement>
+        <requirement type="package" version="3.6">python</requirement>
+        <requirement type="package" version="4.2.2">rgi</requirement>
         <requirement type="package" version="1.68">biopython</requirement>
         <requirement type="package" version="1.10.0">six</requirement>
         <requirement type="package" version="2.6.3">prodigal</requirement>
         <requirement type="package" version="2.6.0">blast</requirement>
         <requirement type="package" version="0.8.36">diamond</requirement>
-        <requirement type="package" version="1.2.8">zlib</requirement>
+		<requirement type="package" version="1.2.8">zlib</requirement>
```

Comment:
It looks like this file has a mixture of tabs and spaces used for indentation. Could you fix up the file so it uses only one type of character (preferably spaces)?

Reply:
It looks like there's still a mixture of spaces and tabs in this file. You may have to enable viewing tabs in the editor you're using to see them.

One command you could use to replace all tabs with spaces is the `sed` command. This will read in a text file, perform operations on it, and print the output back to you (or overwrite the original file).  For example:

```bash
sed -i.bak -e 's/\t/    /g' rgi.xml
```

The following is how to interpret the options:

* `-i.bak`: Perform an operation and overwrite the original file, but first making a backup copy and appending `.bak` to the end of the file name (so you will have a backup named `rgi.xml.bak`).
* `-e 's/\t/    /g'`: The command to run with `sed`
    * `s`: Perform a `s`ubstitution
    * `/\t/`: Find all tab characters
    * `/    /`: Replace all the found tab characters with 4 spaces
    * `/g`: Run this substitute/replace multiple times per line (`g`lobally). Without `g` only the first tab is replaced by spaces in each line.

After running this command you may have to take a look at the file and verify that the XML tabs still line up.

Reply:
These lines have been deleted in the latest commit.

Reply:
Thanks